### DISCUSSION TITLE
Modal header: change default justify-content from space-between to start

### DISCUSSION THREAD

**Discussion by MarkoOleksiyenko** [twbs/bootstrap#39370] [General]
Hello, for the modal header I see that the default justify-content value is set to 'space-between' (https://github.com/twbs/bootstrap/blob/main/scss/_modal.scss#L129).

This makes the content be positioned on the opposite sides of the modal if the 'cross button' is removed and we have composite title with icon, for example.

I see that the button position is actually computed independently in: https://github.com/twbs/bootstrap/blob/main/scss/_modal.scss#L136
So the line with justify-content seems to be useless and it feels like removing it will be better (it should use the justify-content: start as default).

So my question is whether it is a reasonable request :)
Thanks in advance, Marko

**Comment by julien-deramond** [comment]
Hello @MarkoOleksiyenko 
I haven't checked too much in detail the use case but by going back to the history of the file, I can only see that a long time ago, the close button was placed on the right-hand side thanks to that rule and that it's not the case anymore.
The only thing I can relate to this `space-between` positioning is that it's also the default positioning for elements in navbars in Bootstrap.
If this change is done, it must also be done the same way for `.offcanvas-header` which shares the same idea.
Would you mind creating a PR so that we can test it more deeply and gather the thoughts of other maintainers?

**Reply by MarkoOleksiyenko** [reply]
Hello @julien-deramond, sure here it is: https://github.com/twbs/bootstrap/pull/39373
Indeed, checked around and modal and offcanvas are the components that should be modified if we want to have this change.
Please tell me, if you think that we should have examples showing the behavior or if any doc update needed.
